dask/dataframe/core.py

```diff
@@ -115,9 +115,9 @@ def finalize(results):
 class Scalar(DaskMethodsMixin, OperatorMethodMixin):
     """A Dask object to represent a pandas scalar"""
 
-    def __init__(self, dsk, name, meta, divisions=None):
-        # divisions is ignored, only present to be compatible with other
-        # objects.
+    def __init__(self, dsk, name, meta, divisions=None, lens=None):
```

Comment:
If we're going to add this, let's use the full word, "lengths", which I think will be clearer.